src/bindings/jna/libelektra4j/src/main/java/org/libelektra/KeySet.java

```diff
@@ -77,21 +77,30 @@ protected static KeySet create(final int alloc, final Object... args) {
 
 	/**
 	 * Basic constructor for key set
-	 *
-	 * @param alloc
-	 *            Length of key set (key count) to be allocated
-	 * @param args
-	 *            List of initial keys for the key set.
-	 * @return New key set with the given initial data
+     *
+	 * @return New key set with a preallocated space of 16
```

Comment:
The preallocated size is internal and subject to be changed so it should not be documented.

Reply:
Thats true, but even Oracle is doing this so users have a clue about the size at least and if they should set the initial value themselves to avoid unnecessary expandings in the structure:
https://docs.oracle.com/javase/8/docs/api/java/util/ArrayList.html#ArrayList--

I will remove it though if it is your desire :)

Reply:
If Java programmers are used to it, we can provide this information.